src/components/dialogs/parameters/sensi/table-row.tsx

```diff
@@ -29,6 +30,7 @@ const TableRowComponent: FunctionComponent<TableRowComponentProps> = ({
     row,
     index,
     handleDeleteButton,
+    disableDelete,
```

Comment:
could be set false by default. If we can avoid to test false or undefined it's better

Reply:
Done